firebase-sessions/src/main/kotlin/com/google/firebase/sessions/FirebaseSessions.kt

```diff
@@ -41,7 +41,7 @@ class FirebaseSessions internal constructor(firebaseApp: FirebaseApp) {
 
   private fun initiateSessionStart() {
     val sessionState = sessionGenerator.generateNewSession()
-    val sessionEvent = SessionEvent.sessionStart(sessionState)
+    val sessionEvent = SessionEvents.sessionStart(sessionState)
```

Comment:
A couple of minor nitpicks (outside the scope of this PR)

1. Let's rename "sessionState" with "sessionDetails"
2. Can we rename the API `sessionStart` to `startSession`? - Just finding it more readable.

Very minor nitpicks - Feel free to do this in another PR (if that makes it clean for you).

Reply:
1. I missed this in my last PR, fixed!
2. Done. Do you see it like a verb, "do start the session" ? I think I saw it as a noun "this is the session start".

Reply:
yeah! I go with the "Verb on the Noun" making it read like a sentence.